code/modules/paperwork/paper.dm

```diff
@@ -188,9 +187,16 @@
 	else if (has_content)
 		html += PAPER_META("The paper is written in [language.name].")
 		html += "<hr/>" + info
-	html += "[stamps]</body></html>"
-	show_browser(user, html, "window=paper_[name]")
-	onclose(user, "paper_[name]")
+	html += "[stamps]"
+
+	// Ported to browser datum for IE11 feature parity
+	var/datum/browser/window = new(user, "paper_[name]")
+	window.stylesheets.Remove("common")
+	window.add_stylesheet("acs", 'html/acs.css')
```

Comment:
Should probably fix the hacky solution I did of networking the ACS stylesheet for every single piece of paper regardless of whether it uses ACS tags